DrawerController/DrawerController.swift

```diff
@@ -212,7 +212,7 @@ open class DrawerController: UIViewController, UIGestureRecognizerDelegate {
     }
     
     /// View controller
-    public func setViewController(_ viewController: UIViewController?, side: DrawerSide) {
+    public func setVController(_ viewController: UIViewController?, side: DrawerSide) {
```

Comment:
Is it possible to find objc without problems by configuring it like `set(_ controller: UIViewController, side: DrawerSide)`?
`VController` seems to be mismatching Apple's variable name guide because of the reduction.

Reply:
Yeah seems like a good idea, let me try it